src/App.js

```diff
@@ -17,7 +17,30 @@ const styles = theme => ({
     paddingTop: theme.spacing.unit * 20,
   },
 });
-
+class Tile extends React.Component{
+  render(){
+    const tileStyle = {
+      width: '150px',
+      height: '150px',
+      backgroundColor:this.props.color,
+      borderRadius: '5px',
+      boxShadow: '0px 0px 15px 0px',
+      paddingTop:'65px',
+      color: this.props.color,
+      display:'inline-block',
+    };
+    const tileSpanStyle = {
+      color: 'white',
+      fontFamily:'Nunito,sans-serif',
+      fontWeight:700
+  };
+    return (
+      <div style = {tileStyle} >
```

Comment:
I'd worry about indentation if I were you!
`style={tileStyle}`

Anyway, let's not use the style tag and use classes instead. What say?